release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -2845,6 +2845,8 @@ This affects the achievable latency of such clusters and might prevent proper op
 
 * {product-title} 4.12 clusters on the Nutanix platform may have an `Upgradeable=False` condition if they are missing configuration needed for the Nutanix Cloud Control Manager (CCM). To resolve this condition, see: link:https://access.redhat.com/solutions/7014068[How to create the ConfigMaps and Secrets needed to upgrade to OpenShift 4.13 when using Nutanix as a Platform].
 
+* Currently, when using a persistent volume (PV) that contains a very large number of files, the pod might not start or can take an excessive amount of time to start. For more information, see this link:https://access.redhat.com/solutions/6221251[knowledge base article]. Also, for ODF users, see this link:https://access.redhat.com/solutions/6906261[workaround article]. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1987112[*BZ1987112*])
```

Comment:
The ODF known issues does not container the KB#6906261 so please remove in order to be aligned with their release notes. Thanks!

Reply:
Done.